lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -1916,6 +1914,28 @@ prop_updateTx tx extraIns extraCol extraOuts newFee = do
     collateralIns = view (bodyTxL . collateralInputsTxBodyL)
     fee = view (bodyTxL . feeTxBodyL)
 
+prop_mergeValuePositiveNegative_splitValuePositiveNegative
+    :: Pretty Value
+    -> Property
+prop_mergeValuePositiveNegative_splitValuePositiveNegative (Pretty v) =
+    (mergeValuePositiveNegative . splitValuePositiveNegative) v === v
```

Comment:
I think we should add a coverage check here -- what if there are no negative components to the value?

Reply:
Added in https://github.com/cardano-foundation/cardano-wallet/compare/404445b70937c0b663e0e70cbefabebc6d0a211b..6fafa5a1e72ea26b5799814b80b08e827b267979#diff-ccf8f33360a741995f46e00f9cbd19c05009cdfc0132e463eb727c0c6fdde1a1R1955